django_redis/client/default.py

```diff
@@ -507,8 +507,8 @@ def _incr(
                 # In this situations redis will throw ResponseError
 
                 # try to keep TTL of key
+                timeout = self.ttl(key, version=version, client=client)
```

Comment:
Does the sharded client need this too or does it use this function?

Reply:
It uses this function